package.json

```diff
@@ -1,86 +1,73 @@
-        "eslint-plugin-react": "^7.13.0",
-        "eslint-plugin-react-hooks": "^1.6.0",
-        "express": "~4.16.3",
-        "full-icu": "~1.3.1",
-        "husky": "^2.3.0",
-        "jest": "^24.7.0",
-        "lerna": "~2.9.1",
-        "lint-staged": "^8.1.7",
-        "lolex": "~2.3.2",
-        "mutationobserver-shim": "^0.3.3",
-        "prettier": "~1.17.1",
-        "raf": "~3.4.1",
-        "ts-jest": "^24.0.0",
-        "wait-on": "^3.2.0",
-        "whatwg-fetch": "^3.0.0"
-    },
-    "workspaces": [
-        "packages/*",
-        "examples/*",
-        "cypress"
-    ],
-    "dependencies": {
-        "typescript": "^3.5.3"
-    }
+	"private": true,
+	"name": "react-admin-lerna",
+	"scripts": {
+		"build": "lerna run build",
+		"watch": "lerna run --parallel watch",
+		"build-demo": "cd examples/demo && cross-env REACT_APP_DATA_PROVIDER=rest yarn -s build",
+		"test-unit": "cross-env NODE_ENV=test cross-env BABEL_ENV=cjs NODE_ICU_DATA=./node_modules/full-icu jest",
+		"test-unit-ci": "cross-env NODE_ENV=test cross-env BABEL_ENV=cjs NODE_ICU_DATA=./node_modules/full-icu jest --runInBand",
+		"test-e2e": "yarn run -s build && cross-env NODE_ENV=test && cd cypress && yarn -s test",
+		"test-e2e-local": "cd cypress && yarn -s start",
+		"test": "yarn -s test-unit && yarn -s test-e2e",
+		"doc": "cd docs && jekyll server . --watch",
+		"lint": "eslint --ext .js,.ts,.tsx \"./packages/**/src/**/*.{js,ts,tsx}\" \"./examples/**/src/**/*.{js,ts,tsx}\" \"./cypress/**/*.{js,ts,tsx}\"",
+		"prettier": "prettier --config ./.prettierrc.js --write --list-different \"packages/*/src/**/*.{js,json,ts,tsx,css,md}\" \"examples/*/src/**/*.{js,ts,json,tsx,css,md}\" \"cypress/**/*.{js,ts,json,tsx,css,md}\"",
+		"run-simple": "cd examples/simple && yarn -s start",
+		"run-tutorial": "yarn run -s build && cd examples/tutorial && yarn -s start",
+		"run-demo": "cd examples/demo && cross-env REACT_APP_DATA_PROVIDER=rest yarn -s start",
+		"run-graphql-demo": "cd examples/demo && cross-env REACT_APP_DATA_PROVIDER=graphql yarn -s start",
+		"run-demo-watch": "concurrently \"yarn run watch\" \"yarn run run-demo\"",
+		"run-graphql-demo-watch": "concurrently \"yarn run watch\" \"yarn run run-graphql-demo\""
+	},
+	"jest": {
+		"setupFilesAfterEnv": [ "./test-setup.js" ],
+		"preset": "ts-jest/presets/js-with-ts",
+		"testPathIgnorePatterns": [ "/node_modules/", "/lib/", "/esm/", "/examples/simple/" ],
+		"transformIgnorePatterns": [ "[/\\\\]node_modules[/\\\\].+\\.(js|jsx|mjs|ts|tsx)$" ],
+		"globals": {
+			"ts-jest": {
+				"isolatedModules": true
+			}
+		}
+	},
+	"devDependencies": {
+		"@types/jest": "^24.0.13",
+		"@types/react": "^16.9.0",
+		"@types/react-redux": "^7.1.1",
+		"@types/recompose": "^0.27.0",
+		"@typescript-eslint/eslint-plugin": "^1.9.0",
+		"@typescript-eslint/parser": "^1.9.0",
+		"babel-eslint": "^10.0.1",
+		"cheerio": "~1.0.0-rc.2",
+		"concurrently": "^5.1.0",
+		"cross-env": "^5.2.0",
+		"eslint": "6.6.0",
```

Comment:
Please use loose version dependencies (`^`) here and below

Reply:
Done.

Reply:
Done.